README.md

```diff
@@ -1,2 +1,32 @@
-# template-repository
-A template for repositories with standard Strategy Unit files
+# Strategy Unit Branding YAML
+
+A repository containing SU branding and logos in `_brand.yml` file.
+
+`_brand.yml` is [supported](https://posit-dev.github.io/brand-yml/#support) for Quarto and Shiny (R & Python). 
+
+## For Quarto: 
+
+See [Multiformat branding with `_brand.yml`](https://quarto.org/docs/authoring/brand.html#overview) documentation. 
+
+## For R Shiny:
+[bslib](https://rstudio.github.io/bslib/) v0.9.0+ is required to use `_brand.yml`
+
+- For a `_brand.yml` in same directory as Shiny App add `theme = bs_theme(brand = T)` to UI code to include.
```

Comment:
I have to be a [style stickler](https://style.tidyverse.org/syntax.html#logical-vectors) here!

```suggestion
- For a `_brand.yml` in same directory as Shiny App add `theme = bs_theme(brand = TRUE)` to UI code to include.
```